server/delegate/delegate.route.js

```diff
@@ -17,4 +17,9 @@ router
 
   .get(validate(paramValidation.getTopDelegates), topDelegates)
 
+/** GET /api/delegates/rewardStatus/:address - Get delegate reward status*/
+router
+  .route('/rewardStatus/:address')
```

Comment:
Req: The standard best practice for REST APIs is to have a hyphen, not camelcase or underscores. Also It is recommended to use hyphen (which is highlighted by [RFC3986](https://www.ietf.org/rfc/rfc3986.txt)).

Regards

Reply:
Done